app/screens/auth/WalletConnectionType.tsx

```diff
@@ -136,11 +136,14 @@ const WalletConnectionType = ({ history, location }: AuthRouterParams) => {
             <RowText>Setup a wallet that uses a public</RowText>
             <RowText>Spacemesh web service</RowText>
           </RowColumn>
-          <Button
-            text="WALLET ONLY"
-            width={150}
-            onClick={handleNextStep(true)}
-          />
+          <div title="Temporarily unavailable">
```

Comment:
For me ok, maybe we want to add an additional message in gray color

<img width="314" alt="Screenshot 2023-07-11 at 15 42 18" src="https://github.com/spacemeshos/smapp/assets/54288612/52003787-51ab-42e6-bb0a-e750d695bc67">